macaroonbakery/httpbakery/agent.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2017 Canonical Ltd.
+# Licensed under the LGPLv3, see LICENCE file for details.
+
+import base64
+import json
+
+import nacl.public
+import nacl.encoding
+import requests.cookies
+import six
+from six.moves.urllib.parse import urlparse
+
+
+def load_agent_file(filename):
+    """ load_agent_file loads agent information from the specified file.
+
+        The agent information loaded is returned as a cookie jar and a key,
+        these can be passed directly as the cookies and key parameter to
+        BakeryAuth.
+    """
+
+    with open(filename) as f:
```

Comment:
open(filename, cookies) so we can just add a cookie to the existing one if someone has a set of necessary cookie for its request ?